recipes/python-toolchain/bld.bat

```diff
@@ -0,0 +1,16 @@
+setlocal EnableDelayedExpansion
+
+:: This works for `setuptools`, but breaks `distutils`.
+:: Will need to figure out a better long term strategy.
+copy "%RECIPE_DIR%\distutils.cfg" "%STDLIB_DIR%\distutils\distutils.cfg"
+
+:: Configure `pip`.
+IF NOT EXIST "%LIBRARY_PREFIX%\etc" mkdir "%LIBRARY_PREFIX%\etc"
+copy "%RECIPE_DIR%\pip.conf" "%LIBRARY_PREFIX%\etc\pip.conf"
+
+:: Copy the [de]activate scripts to %LIBRARY_PREFIX%\etc\conda\[de]activate.d.
+:: This will allow them to be run on environment activation.
+FOR %%F IN (activate deactivate) DO (
+    IF NOT EXIST %LIBRARY_PREFIX%\etc\conda\%%F.d MKDIR %LIBRARY_PREFIX%\etc\conda\%%F.d
+    COPY %RECIPE_DIR%/%%F.bat %LIBRARY_PREFIX%\etc\conda\%%F.d\python-toolchain_%%F.bat
+)
```

Comment:
I  currently use this (without delayed expansion) in my go port (not yet pushed):

```bat
for %%F in (activate deactivate) do (
    if not exist "%LIBRARY_PREFIX%\etc\conda\%%F.d" mkdir "%LIBRARY_PREFIX%\etc\conda\%%F.d"
    copy "%RECIPE_DIR%\%%F.bat" "%LIBRARY_PREFIX%\etc\conda\%%F.d\go_%%F.bat"
)
if errorlevel 1 exit 1
```
-> different slashes in the copy line